pytorch_lightning/trainer/trainer.py

```diff
@@ -822,6 +828,28 @@ def slurm_job_id(self) -> int:
             job_id = None
         return job_id
 
+    @classmethod
+    def default_attributes(cls):
+        return vars(cls())
+
+    @classmethod
+    def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
+        """Extend existing argparse by default `Trainer` attributes."""
+        parser = ArgumentParser(parents=[parent_parser])
+
+        trainer_default_params = Trainer.default_attributes()
+
+        for arg in trainer_default_params:
+            parser.add_argument('--{0}'.format(arg), default=trainer_default_params[arg], dest=arg)
+
+        return parser
+
+    @classmethod
+    def from_argparse_args(cls, args):
+
```

Comment:
shouldn't there be documentation where the blank line is? did it get lost?

Reply:
yeah it should...